utils/wsutil/op.go

```diff
@@ -17,12 +17,12 @@ var ErrEmptyPayload = errors.New("empty payload")
 type OPCode uint8
 
 type OP struct {
-	Code OPCode   `json:"op"`
-	Data json.Raw `json:"d,omitempty"`
-
 	// Only for Gateway Dispatch (op 0)
-	Sequence  int64  `json:"s,omitempty"`
-	EventName string `json:"t,omitempty"`
+	EventName string   `json:"t,omitempty"`
+	Data      json.Raw `json:"d,omitempty"`
+	// Only for Gateway Dispatch (op 0)
+	Sequence int64  `json:"s,omitempty"`
```

Comment:
The `Only for Gateway Dispatch (op 0)` references both `Sequence` and `EventName`.
Change `Sequence`'s comment to use the godoc convention and add a comment to `EventName`.